src/system/game-data.ts

```diff
@@ -917,186 +903,177 @@ export class GameData {
     if (response) {
       return false;
     }
-    localStorage.setItem(`sessionData${slotId ? slotId : ""}_${loggedInUser?.username}`, encrypted);
+    localStorage.setItem(getSaveDataLocalStorageKey(slotId), encrypted);
     const success = await updateUserInfo();
     return !(success !== null && !success);
   }
 
-  async loadSession(slotId: number, sessionData?: SessionSaveData): Promise<boolean> {
-    const { promise, resolve, reject } = Promise.withResolvers<boolean>();
-    try {
-      const initSessionFromData = (fromSession: SessionSaveData) => {
-        if (isLocal || isBeta) {
-          try {
-            console.debug(
-              this.parseSessionData(
-                JSON.stringify(fromSession, (_, v: any) => (typeof v === "bigint" ? v.toString() : v)),
-              ),
-            );
-          } catch (err) {
-            console.debug("Attempt to log session data failed:", err);
-          }
-        }
+  /**
+   * Load stored session data and re-initialize the game with its contents.
+   * @param slotIndex - The 0-indexed position of the save slot to load.
+   * Values `<=0` will be considered invalid.
```

Comment:
Either indent the second line or merge them.
```suggestion
   * @param slotIndex - The 0-indexed position of the save slot to load. Values `<=0` will be considered invalid.
```